### PR TITLE
Mobile menu

### PR DESCRIPTION
**Pull request by Daphineatim**
- Fixed linters
-  Deleted linters
-  Updated README.md file
-  Deleted workflows and added linters
-  Fixed linter errors

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -30,7 +30,7 @@ <h3><a href="/"></a>My Logo</h3>
       <section class="bg-color-1">
         <div class="bg-image-1"></div>
         <div class="headline">
-          <h1 class="headline-inside">I’m Daphine Glad to see you!</h1>
+          <h2 class="headline-inside">I’m Daphine Glad to see you!</h2>
```
Comment (cynthiainga) [inline index.html:33]: - [ ] The main title (e.g., "I'm Daphine ...") is a `h1` tag. If you want that google (and other search engines) know about a specific topic, in this case, you, use an `h1` tag. [Check this link](https://www.reliablesoft.net/h1-tag/) for more info. If you want that google (and other search engines) know about a specific topic, in this case, you, use an h1 tag. [Check this link](https://www.reliablesoft.net/h1-tag/) for more info 🙃 🙃

File: index.html [hunk 2]
```diff
@@ -30,7 +30,7 @@ <h3><a href="/"></a>My Logo</h3>
...
           <p class="headline-inside2">
             I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If
             you like what you see and have a project you need coded, don’t hestiate to contact me.
```
Comment (cynthiainga) [inline index.html:36]: - [ ] So as I can't select the specific line of codes, I select this part. Just after this part, the list of icons uses a `ul` tag to group them, and `li` per icon; each icon has a link (`a` tag). Apart from being semantically correct, the `a` tag will let you connect your social networks with the icons, using href, please kindly add your `img` tag in the `a` tag 🙃 🙃

**Review by cynthiainga — CHANGES_REQUESTED** [review]
Hi @Daphineatim, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


## Highlights

- [ ] No linters errors ✔️ 
- [ ] Good commit history ✔️ 
- [ ] Good design 👍 

## Required Changes ♻️

- [ ] Pull Request should have a short summary describing introduced changes in more detail, please kindly add it 🙃 🙃
- [ ] For other changes required please check the comments under the review 👇 👇

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by sumairq — APPROVED** [review]
### Hi @Daphineatim  , 



Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
✔️ Good commit history
✔️Linters are passing
✔️Very professional README
✔️Correct Github flow
✔️Descriptive PR
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**